index.js

```diff
@@ -0,0 +1,77 @@
+const checkUpperCase = (letter) => {
+  let hasUpperCase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+
+  for (let i = 0; i < hasUpperCase.length; i++) {
+    const currentItem = hasUpperCase[i]
+
+    if (currentItem === letter) return true
+  }
+
+  return false
+}
+
+const checkLowerCase = (letter) => {
+  let hasLowerCase = 'abcdefghijklmnopqrstuvwxyz'
+
+  for (let i = 0; i < hasLowerCase.length; i++) {
+    const currentItem = hasLowerCase[i]
+
+    if (currentItem === letter) return true
+  }
+
+  return false
+}
+
+const checkNumbers = (letter) => {
+  let hasNumber = '1234567890'
+
+  for (let i = 0; i < hasNumber.length; i++) {
+    const currentItem = hasNumber[i]
+
+    if (currentItem === letter) return true
+  }
+
+  return false
+}
+
+const checkSpecCharacter = (letter) => {
+  let hasSpecial = '!@#$%^&*()_-+='
+
+  for (let i = 0; i < hasSpecial.length; i++) {
+    const currentItem = hasSpecial[i]
+
+    if (currentItem === letter) return true
+  }
+
+  return false
+}
+
+
+const validatePassword = (password) => {
+  let hasLowerCase = false
+  let hasNumber = false
+  let hasSpecial = false
+  let hasUpperCase = false
+
+  for (let i = 0; i < password.length; i++) {
+    if (checkUpperCase(password[i]) === true) {
+      hasUpperCase = true
+    }
```

Comment:
The if/else statement executes a block of code if a specified condition is true. Removing === true will give you the same result.